app/build.gradle.kts

```diff
@@ -210,6 +210,7 @@ dependencies {
     implementation("com.github.AppDevNext:ChangeLog:3.4")
 
     "fullImplementation"("androidx.car.app:app:1.3.0-rc01")
+    implementation("androidx.car.app:app-projected:1.2.0")
```

Comment:
this should also use the `fullImplementation` as these sensors are not offered in the minimal flavor and we need to make sure no google service dependency is added to that flavor.